cy_scripts/snippet-creator/cli.js

```diff
@@ -0,0 +1,107 @@
+const program = require('commander')
+const chalk = require('chalk')
+const fs = require('fs-extra')
+const debug = require('debug')('snippet')
+const path = require('path')
+const Confirm = require('prompt-confirm')
+
+
+const convert = require('./convert')
+
+global.Promise = require('bluebird')
```

Comment:
This is almost never a good idea to do. it's generally better to always import it. You could be overriding the way stuff works in code that's not yours that's using native promises. Although bluebird is technically compliant with A+ Promises, it's just not a good idea.